deploy/manifests/lvmd/lvmd.yaml

```
@@ -0,0 +1,70 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: topolvm-system
+  name: topolvm-lvmd
```

```suggestion
  name: lvmd
```

The ServiceAccount name for node is `node` and controller is `controller`. `lvmd` is better if we follow it.

---

fixed  it.
https://github.com/topolvm/topolvm/pull/285/commits/2f348e042a1885c1b417cd68e2f8f955893fcc26